ddl.dic

```diff
@@ -715,6 +715,199 @@ save_dictionary_audit.version
 
 save_
 
+save_DICTIONARY_AUTHOR
+
+    _definition.id                DICTIONARY_AUTHOR
+    _definition.scope             Category
+    _definition.class             Loop
+    _definition.update            2023-06-02
+    _description.text
+;
+    The CATEGORY of data items used for dictionary author(s) details.
```

Comment:
```suggestion
    Attributes used to record the dictionary author information.
```

Reply:
Minor rewording to make it more similar to other category definitions in this dictionary (and the `PUBL_AUTHOR` definition).